src/app/shared/kng-user-reminder/kng-user-reminder.component.ts

```diff
@@ -62,46 +65,32 @@ export class KngUserReminderComponent implements OnInit {
     private $snack: MdcSnackbar
   ) {
     const loader = this.$route.snapshot.parent.data['loader'] || this.$route.snapshot.data['loader'];
+    this.selectedNotification = this.weekdays[this.weekdays.length - 1];
+
     if (loader.length) {
       this.user = this.user || loader[1];
+      this.selectedNotification = this.user.reminder.weekdays[0];
     }
-  }
+   }
 
-  doUpdate(event, day: number, time) {
-    // FIXME change envent called 2x!
-    // console.log('MdcCheckboxChange',event.type)
-    if (!event.type) {
-      return;
-    }
+  doUpdate(event) {
+    let day = event.value.value;
 
     if (day) {
       day = day | 0;
-      let pos = this.user.reminder.weekdays.indexOf(day);
-      if (pos === -1) {
-        this.user.reminder.weekdays.push(day);
-      } else {
-        do {
-          this.user.reminder.weekdays.splice(pos, 1);
-          pos = this.user.reminder.weekdays.indexOf(day);
-        }while (pos > -1);
-
-      }
-    }
-    if (time) {
-      time = time | 0;
-      this.user.reminder.time = time;
+      this.user.reminder.weekdays = [];
+      this.user.reminder.weekdays.push(event.value.value);
+      this.user.reminder.time = this.time;
     }
     this.user.reminder.active = !!(this.user.reminder.weekdays.length);
```

Comment:
ya un bug ici, tu as tjrs un weekday puisque 'Sans notification' c'est le day 0. 
il vaut mieux faire un truc ou `'Sans notification' => day = -1 ET quand day == -1 weekday=[] `

Reply:
...
{value: '-1', label: 'Sans notification'}
...
if (event.value.value > 0) {
        this.user.reminder.weekdays.push(event.value.value);
      }

Reply:
@nath-0-0 Nein!
Je dirais plutot:
```js
this.user.reminder.weekdays=(event.value>-1)?[event.value]:[];
```